.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL=https://parity0.kovan.makerfoundation.com
```

Comment:
we can remove this URL

Reply:
Yep ok, was trying to make it plug and play